### DISCUSSION TITLE
Elastalert2 notification card structure for Teams Power Automate

### DISCUSSION THREAD

**Discussion by MikeeAndrew** [jertel/elastalert2#1552] [Q&A] [answered]
Hello, I am using the 2.20.0 release with Teams workflow (Power Automate) to post a card in a channel when a webhook request is received.
The problem is that the card I receive has an awkward structure and I would like to make it more readable.
As you can see, I have two columns that interest me: container_name and count, but they and their values are not well formatted.
Could someone please help me understand how to modify the notification card?
(The values of the container names are the ones covered in black)

 
![immagine](https://github.com/user-attachments/assets/0e6be2fd-c0b5-4e19-bbeb-81cf2cb6307f)

**Comment by jertel** [comment]
I see this in the [docs](https://elastalert2.readthedocs.io/en/latest/alerts.html#microsoft-teams), it might help:

![image](https://github.com/user-attachments/assets/758dbc96-04cf-4e05-addf-14cb6bdc6624)

**Reply by MikeeAndrew** [reply]
Hi jertel, thanks for the answer.
Sorry for the confusion. I am using Power Automate since Microsoft will deprecate the Office 365 connectors. 
I found the property "ms_power_automate_teams_card_width_full" in the documentation, but setting it to true made it even worse.
I also tried to mix it with different body text sizes with the property "ms_power_automate_body_text_size", but no results satisfied me.
The standard elastalert2 notification is good-looking from the phone, but it's a different story from a laptop.
I would like to get rid of all those hyphens and other signs, and maybe do it like an unordered list or something similar.
Is there a template I can modify?

**Reply by jertel** [reply, marked as answer]
Have you tried changing the [summary_table_type](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#summary-table-type) setting?

**Reply by MikeeAndrew** [reply]
Hi jertel, setting summary_table_type to markdown solved the issue.
Thanks a lot for the support provided!
